### DISCUSSION TITLE
训练自己的数据集时出现问题

### DISCUSSION THREAD

**Discussion by Detectiveyw** [secretflow/secretflow#1430] [Q&A]
训练数据集代码如下
<img width="1057" alt="86af1be0c88370fef79e61c5b9971d6" src="https://github.com/user-attachments/assets/f5587aeb-32f9-43a2-b781-5bc768f1f14b">
出现的问题：
<img width="1046" alt="01374789c2a8b5f7fdd956f9bdd77f7" src="https://github.com/user-attachments/assets/f59e8b03-e847-44a8-b544-3d88d919c91f">
<img width="1060" alt="907e57f9d864349d6a918da6f45ea3f" src="https://github.com/user-attachments/assets/4f858f1e-1f48-4379-bcb7-94dd674e9847">
<img width="1051" alt="4ad553c45b2dd343effa0811f6a24d6" src="https://github.com/user-attachments/assets/087bdd59-3cd9-4833-8736-36c333b43e4b">

**Comment by 383004576** [comment]
您好，我们可能需要您提供详细的版本环境以及复现步骤来帮助定位问题，麻烦请移步issue区提单：https://github.com/secretflow/secretflow/issues
